packages/neuron-ui/src/widgets/InputSelect/index.tsx

```diff
@@ -122,7 +125,7 @@ const Select = ({ value, options, placeholder, disabled, onChange, className }:
         data-open={openRef.current}
       >
         <input className={placeholderClass} onChange={onInputChange} value={value ?? innerValue} />
-        <div className={styles.arrow} />
+        {isNewUI ? <Arrow className={styles.arrowForNewUI} /> : <div className={styles.arrow} />}
       </div>
       {openRef.current ? (
```

Comment:
I don't think `isNewUI ` is needed since we will be using the new UI design

Reply:
same above.